### ISSUE TITLE
Does not work on HW 2.72

### ISSUE THREAD

**Issue by PetteriAimonen** [PetteriAimonen/dso-quad-usb-analyzer#1]
>    Oh noes: FPGA config byte fifo underflow
> Caller 0802928b
> Callers 080291ef 0802928b 08025ba1 08029241
> Memory dump saved to memory.dmp.

New 8MB flash is too slow to read. Have to find a workaround.

**Comment by es4** [comment]
I get also this message, if I start the "locic capture" sw on the DSO203 with the HW 2.72.
Did you find a asolution to this problem?

**Comment by PetteriAimonen** [comment]
Nope, I have no interest in trying to fix it. Works for my HW, cannot test on others.
